src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
 // write your code here
+const elements = document.getElementsByClassName('population');
+
+const populations = Array.from(elements).map((element) => {
+  return parseInt(element.innerText.replace(/,/g, ''), 10);
+});
+
+const averagePopulation = document.querySelector('.average-population');
+
+averagePopulation.innerText = averagePeople(populations);
+
+const totalPopulation = document.querySelector('.total-population');
+
+totalPopulation.innerText = totalPople(populations);
```

Comment:
There is a typo in the function name `totalPople`. It should be `totalPeople` to match the naming convention and avoid confusion.